pyshelf/configure.py

```diff
@@ -14,8 +14,12 @@ def app(app):
             "accessKey": val.get("accessKey"),
```

Comment:
I think we need to fix this.  Since the change to contain multiple bucket names, it seems this code doesn't make sense anymore.  We should be validating that ALL buckets have these.

Also, we should probably nest the buckets under a "buckets" property in case we add any other configuration (for example "elasticSearchHost"